docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,989 @@
+---
```

Comment:
you need to add this to mkdocs.yaml otherwise it doesn't show up https://deploy-preview-1886--hypershift-docs.netlify.app/how-to/